acceptance/snaps/snaps.go

```diff
@@ -95,7 +95,15 @@ func MatchSnapshot(ctx context.Context, qualifier, text string, vars map[string]
 	errs := capture(ctx, qualifier)
 
 	for k, v := range vars {
-		text = strings.ReplaceAll(text, v, "${"+k+"}")
+		// Todo: remove ASAP. Just a hack until a solution is figured out
+		if k == "IMAGE_SIGNATURES_XML_acceptance/image" {
```

Comment:
As mentioned on slack, workaround/hack is okay to merge, but I think there should be a note to help explain what it does and why it's needed.